wonkyung/.env.sample

```
@@ -0,0 +1,16 @@
+.env
+node_modules/
+
+
+DATABASE_URL="mysql:/username:password@127.0.0.1:3306/DATABASE_name"
```

```suggestion
DATABASE_URL="rdbms:/username:password@host:port/database-name"
```

1. 우선 mysql 데이터베이스를 직접 명시하지 않아도 됩니다.
2. host와 port 역시 직접 명시하지 않아도 됩니다.
3. url 구조의 통일성을 위해 대소문자를 지켜주면 좋을 것 같습니다.